modules/rosa-configure.adoc

```diff
@@ -13,6 +13,11 @@ Use the following commands to configure the {product-title} (ROSA) CLI, `rosa`.
 
 Log in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token from link:https://console.redhat.com/openshift/token/rosa[the ROSA token page].
 
+[NOTE]
```

Comment:
Not accurate. This instruction must go when Proxy is configured with ROSA CREATE CLUSTER and ROSA EDIT CLUSTER commands